### DISCUSSION TITLE
Hello, looking to give credit where its due!

### DISCUSSION THREAD

**Discussion by mitchellgilmore23** [hmes98318/r6s-stats-api#16] [Show and tell]
Hi hmes98318, i'm not sure how to contact you, and i'm also not super familiar with how github works yet completely. With that said, I have cloned your repo here and made changes to it, compiled it, and put it in my own repository.  https://github.com/mitchellgilmore23/Tracker-gg. Just wanted to give you a heads up and wanting to know if there's anything I need to do to give you credit. 
![2023-02-25_12-40-20](https://user-images.githubusercontent.com/117942362/221388225-b89ec562-5de3-476e-bddb-0b34b5717abb.jpg)

**Comment by hmes98318** [comment]
Hi @mitchellgilmore23 , thanks for letting me know that you've cloned my repository, made some changes to it, and put it in your own repository on GitHub. I appreciate your interest in my work!

Regarding your question about giving credit, simply attributing the original source (i.e., my repository) in your repository's README file or documentation would be sufficient. You can do this by including a brief note or hyperlink that points to my repository.

If you have any further questions or need assistance, feel free to contact me at **hmes98318@gmail.com**.

Thank you again for your interest in my project!


Best regards,
hmes98318

**Reply by mitchellgilmore23** [reply]
Got it!
